.github/workflows/drivers_stage1.yaml

```diff
@@ -10,14 +10,16 @@ name: gefs_osn
 
 jobs:
   docker:
-    runs-on: [self-hosted]
+    runs-on: [self-hosted] # do we have access to this in usgsrc4cast repo?
```

Comment:
@rqthomas  do we have access to this runner in the USGS challenge repo? Based on neon4cast-ci jobs, looks like the runner name is `efi-runner-cirrus` ?

Reply:
The runner is an organizational wide runner so it should work.  @cboettig can look at it.

Reply:
I don't see any self-hosted runners configured in the repo https://github.com/eco4cast/usgsrc4cast-ci/settings/actions/runners , and I also can't look at the settings of `neon4cast-ci` to see what it should look like because I don't have permissions.